Templates/Lasso/Screen.xctemplate/ScreenModule/___FILEBASENAME___Store.swift

```diff
@@ -0,0 +1,18 @@
+//___FILEHEADER___
+
+import Foundation
+import Lasso
+
+final class ___VARIABLE_name___Store: LassoStore<___VARIABLE_name___Screen> {
+    
+    // MARK: Init
+    
+    required init(with initialState: State) {
+        super.init(with: initialState)
+    }
+    
+    // MARK: Actions
+    
+    override func handleAction(_ action: Action) {}
```

Comment:
Maybe insert an empty `switch` here (which will force filling it in)?
```suggestion
    override func handleAction(_ action: Action) {
        switch action {
        }
    }
```